<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-10T16:35:00 IST" name="Suite" finished-at="2024-02-10T16:35:43 IST" duration-ms="42514">
    <groups>
    </groups>
    <test started-at="2024-02-10T16:35:00 IST" name="Test-1" finished-at="2024-02-10T16:35:12 IST" duration-ms="11652">
      <class name="rahulshettyacademy.TestDemo">
        <test-method signature="titleCheck()[pri:0, instance:rahulshettyacademy.TestDemo@de88ac6]" started-at="2024-02-10T16:35:00 IST" name="titleCheck" finished-at="2024-02-10T16:35:10 IST" duration-ms="10027" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at rahulshettyacademy.TestDemo.titleCheck(TestDemo.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheck -->
      </class> <!-- rahulshettyacademy.TestDemo -->
    </test> <!-- Test-1 -->
    <test started-at="2024-02-10T16:35:00 IST" name="Test-0" finished-at="2024-02-10T16:35:27 IST" duration-ms="26851">
      <class name="rahulshettyacademy.TestDemo">
        <test-method signature="titleCheck()[pri:0, instance:rahulshettyacademy.TestDemo@2822c6ff]" started-at="2024-02-10T16:35:00 IST" name="titleCheck" finished-at="2024-02-10T16:35:26 IST" duration-ms="25245" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at rahulshettyacademy.TestDemo.titleCheck(TestDemo.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheck -->
      </class> <!-- rahulshettyacademy.TestDemo -->
    </test> <!-- Test-0 -->
    <test started-at="2024-02-10T16:35:14 IST" name="Test2-0" finished-at="2024-02-10T16:35:39 IST" duration-ms="24952">
      <class name="rahulshettyacademy.TestDemo2">
        <test-method signature="titleCheckGoogle()[pri:0, instance:rahulshettyacademy.TestDemo2@4a0df195]" started-at="2024-02-10T16:35:14 IST" name="titleCheckGoogle" finished-at="2024-02-10T16:35:38 IST" duration-ms="23361" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheckGoogle -->
      </class> <!-- rahulshettyacademy.TestDemo2 -->
    </test> <!-- Test2-0 -->
    <test started-at="2024-02-10T16:35:29 IST" name="Test2-1" finished-at="2024-02-10T16:35:43 IST" duration-ms="13301">
      <class name="rahulshettyacademy.TestDemo2">
        <test-method signature="titleCheckGoogle()[pri:0, instance:rahulshettyacademy.TestDemo2@5da7cee2]" started-at="2024-02-10T16:35:29 IST" name="titleCheckGoogle" finished-at="2024-02-10T16:35:41 IST" duration-ms="11774" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheckGoogle -->
      </class> <!-- rahulshettyacademy.TestDemo2 -->
    </test> <!-- Test2-1 -->
  </suite> <!-- Suite -->
</testng-results>
